script/age_grouping.R

```diff
@@ -0,0 +1,83 @@
+## Age Grouping
+#Preparations
+library(tidyverse)
+
+clean_data <- read.csv(here::here("I:/pwind/Nextcloud/Datascience/OceanNow_data/clean_data.csv"))
+
+# Inspect the age variable
+names(clean_data)
+clean_data <- mutate(Age = 2021 - BirthYear, clean_data)
+
+dim(clean_data)
+length(clean_data$Age)
+
+table(clean_data$Age, useNA = "always")
+quantile(clean_data$Age)
+quantile(clean_data$Age, probs = seq(0,1,1/3)) # With 1/3 steps
+
+his <- ggplot(clean_data, aes(x = Age)) +
+  geom_histogram()
+print(his)
```

Comment:
R applies the print function by default when you call an object, so you don't need `print()` here.
```suggestion
his
```